src/main/java/Kayo/Kayo.java

```diff
@@ -0,0 +1,317 @@
+        /**
+         * shows total tasks in format
+         */
+        public void showTotalTasks(List<Task> listTasks) {
+            System.out.print("Now you have " + listTasks.size() + " tasks in the list.\n");
+        }
+        /**
+         * displays deleted task
+         */
+        public void deleteTask(Task task) {
+            System.out.print("Noted. I've removed this task: \n");
+            System.out.print(task + "\n");
+        }
+        public void markTask(Task task) {
+            System.out.print("Nice! I've marked this task at done: \n");
+            System.out.print(task + "\n");
+        }
+        public void unmarkTask(Task task){
+            System.out.print("OK, I've marked this task as not done yet: \n");
+            System.out.print(task + "\n");
+        }
+        public void listItems(List<Task> listTasks) {
+            System.out.println("Here are the tasks in your list:");
+            for(int i = 0; i < listTasks.size(); i++) {
+                System.out.println(i+1 + ". "+ listTasks.get(i));
+            }
+        }
+        public void greet() {
+            System.out.println("Hello! I'm Kayo! ");
+            System.out.println("What can I do for you?");
+        }
+        public void exit(){
+            System.out.println("Bye. I hope to see you again soon!");
+        }
+    }
+    /**
+     * Handles the storage and access of data
+     * **/
+    public static class Storage{
+        public String filepath;
+        /**
+         * Handles the update of data
+         * @param listTasks list of tasks to add to data
+         * */
+        private static void updateData(List<Task> listTasks) {
+            try {
+                FileWriter fw = new FileWriter("data/kayo.txt");
+                String dataBody = "";
+                for (int i = 0; i < listTasks.size(); i++) {
+                    dataBody += listTasks.get(i).toString() + System.lineSeparator();
+                }
+                fw.write(dataBody);
+                fw.close();
+            } catch (IOException e) {
+                System.out.println("File not found");
+            }
+        }
+
+        public Storage(String filepath){
+            this.filepath = filepath;
+        }
+        /**
+         * Loads data from database
+         * @return List of tasks from database
+         * */
+        public List<Task> load () {
+            List<Task> listTasks = new ArrayList<>();
+            try {
+                File f = new File(filepath);
+                Scanner sc = new Scanner(f);
+                while (sc.hasNextLine()) {
+                    String readString = sc.nextLine();
+                    Task task = getTask(readString);
+                    listTasks.add(task);
+                }
+            } catch (FileNotFoundException e) {
+                System.out.println("File not found");
+            }
+            return listTasks;
+        }
+        private Task getTask(String readString) {
```

Comment:
good job on naming all methods correctly!

Reply:
Thank you!